codegen/template_bundle/template_files.go

```diff
@@ -3767,31 +3767,31 @@ type bintree struct {
 }
 
 var _bintree = &bintree{nil, map[string]*bintree{
-	"augmented_mock.tmpl":                {augmented_mockTmpl, map[string]*bintree{}},
-	"dependency_struct.tmpl":             {dependency_structTmpl, map[string]*bintree{}},
-	"endpoint.tmpl":                      {endpointTmpl, map[string]*bintree{}},
-	"endpoint_collection.tmpl":           {endpoint_collectionTmpl, map[string]*bintree{}},
-	"endpoint_test.tmpl":                 {endpoint_testTmpl, map[string]*bintree{}},
-	"endpoint_test_tchannel_client.tmpl": {endpoint_test_tchannel_clientTmpl, map[string]*bintree{}},
-	"fixture_types.tmpl":                 {fixture_typesTmpl, map[string]*bintree{}},
-	"grpc_client.tmpl":                   {grpc_clientTmpl, map[string]*bintree{}},
-	"http_client.tmpl":                   {http_clientTmpl, map[string]*bintree{}},
-	"main.tmpl":                          {mainTmpl, map[string]*bintree{}},
-	"main_test.tmpl":                     {main_testTmpl, map[string]*bintree{}},
-	"middleware_http.tmpl":               {middleware_httpTmpl, map[string]*bintree{}},
-	"middleware_tchannel.tmpl":           {middleware_tchannelTmpl, map[string]*bintree{}},
-	"module_class_initializer.tmpl":      {module_class_initializerTmpl, map[string]*bintree{}},
-	"module_initializer.tmpl":            {module_initializerTmpl, map[string]*bintree{}},
-	"module_mock_initializer.tmpl":       {module_mock_initializerTmpl, map[string]*bintree{}},
-	"service.tmpl":                       {serviceTmpl, map[string]*bintree{}},
-	"service_mock.tmpl":                  {service_mockTmpl, map[string]*bintree{}},
-	"structs.tmpl":                       {structsTmpl, map[string]*bintree{}},
-	"tchannel_client.tmpl":               {tchannel_clientTmpl, map[string]*bintree{}},
-	"tchannel_client_test_server.tmpl":   {tchannel_client_test_serverTmpl, map[string]*bintree{}},
-	"tchannel_endpoint.tmpl":             {tchannel_endpointTmpl, map[string]*bintree{}},
-	"workflow.tmpl":                      {workflowTmpl, map[string]*bintree{}},
-	"workflow_mock.tmpl":                 {workflow_mockTmpl, map[string]*bintree{}},
-	"workflow_mock_clients_type.tmpl":    {workflow_mock_clients_typeTmpl, map[string]*bintree{}},
+	"augmented_mock.tmpl":                &bintree{augmented_mockTmpl, map[string]*bintree{}},
```

Comment:
I don't understand this change -- did goimports make this change?

Reply:
Yes, goimports on this file makes this change.